FEM/fem_ele_vec.cpp

```diff
@@ -1721,7 +1683,7 @@ void CFiniteElementVec::GlobalAssembly_RHS()
 							{
 								const double p0 = h_pcs->GetNodeValue(nodes[i], idx_p1_ini);
 								const double Sat0 = LoadFactor * m_mmp->SaturationCapillaryPressureFunction(-p0);
-								nodal_pore_p[i] -= (p0 > 0.0) ? LoadFactor * Sat0 * p0 : 0.0;
```

Comment:
removed the restriction of positive pore pressure in the total stress calculation for the partially saturated flow.